www/include/RunResultHtmlTable.php

```diff
@@ -17,7 +17,7 @@ class RunResultHtmlTable {
   const COL_RESULT = "result";
   const COL_COST = "cost";
   const COL_CERTIFICATE_BYTES = "certificate_bytes";
-  const COL_FIRST_CONTENTFUL_PAINT = 'PerformancePaintTiming.first-contentful-paint';
+  const COL_FIRST_CONTENTFUL_PAINT = 'FirstContentfulPaint';
```

Comment:
Lowercase f?

Reply:
You're too fast Meenan! :) Yeah..I fixed this.